Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/core/PersonCertificates.kt

```diff
@@ -16,7 +16,7 @@ data class PersonCertificates(
     }
 
     // TODO: would be implemented in admission state logic task
-    val admissionState = AdmissionState.ThreeGWithPCR(highestPriorityCertificate!!)
+    val admissionState: AdmissionState = AdmissionState.ThreeGWithPCR(highestPriorityCertificate!!)
```

Comment:
This will probably get changed anyway, but why do we need to specify the return type explicitly?

Reply:
Because otherwise I could not use it in view model, as the type would be ThreeGWithPCR and not AdmissionState.